File: Funciones/5. promedio_notas/js/promedio.js
/**
 * Promedio de tres notas
 * Autor: Santiago GM
 * 02/04/2024
*/
let numUno;
let numDos;
let numTres;

//como paramétro
function promedios(pnumUno,pnumDos,pnumTres){

    numUno = pnumUno;
    numDos = pnumDos;
    numTres = pnumTres;

    let promedio;
    promedio = (numUno + numDos + numTres)/3;
    return  promedio; 
}
//Como Expresión
const promediosExp= function (pnumUno,pnumDos,pnumTres){

    numUno = pnumUno;
    numDos = pnumDos;
    numTres = pnumTres;

    let promedio;
    promedio = (numUno + numDos + numTres)/3;
    return  promedio; 
}

File: python/5. promedio_notas/utils/promedio.py
# Promedio de tres notas
# Autor: Santiago GM


# Como paramétro
def calcular_promedio(nota_p, porcentaje_p):
    nota, porcentaje  = nota_p, porcentaje_p
    resultado = nota * porcentaje
    return resultado

# Como Expresión
calcular_promedio_exp = lambda nota,porcentaje: nota * porcentaje